### PR TITLE
BZ2056385: Adds mirror registry example

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.9+

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2056385

Link to docs preview: http://file.rdu.redhat.com/sdudhgao/mirror-reg-config/openshift_images/image-configuration.html#images-configuration-registry-mirror_image-configuration

### CODE REVIEW

File: modules/images-configuration-registry-mirror.adoc [hunk 1]
```diff
@@ -114,12 +129,12 @@ NAME                           STATUS                     ROLES    AGE  VERSION
 ip-10-0-137-44.ec2.internal    Ready                      worker   7m   v1.24.0
 ip-10-0-138-148.ec2.internal   Ready                      master   11m  v1.24.0
 ip-10-0-139-122.ec2.internal   Ready                      master   11m  v1.24.0
-ip-10-0-147-35.ec2.internal    Ready,SchedulingDisabled   worker   7m   v1.24.0
+ip-10-0-147-35.ec2.internal    Ready   worker   7m   v1.24.0
```
Comment (xiuwang) [inline modules/images-configuration-registry-mirror.adoc:132]: Need to be aligned

File: modules/images-configuration-registry-mirror.adoc [hunk 2]
```diff
@@ -114,12 +129,12 @@ NAME                           STATUS                     ROLES    AGE  VERSION
...
 ip-10-0-153-12.ec2.internal    Ready                      worker   7m   v1.24.0
 ip-10-0-154-10.ec2.internal    Ready                      master   11m  v1.24.0
 ----
 +
-You can see that scheduling on each worker node is disabled as the change is being applied.
+You can see that scheduling on each worker node is disabled as the change is being applied. The `Imagecontentsourcepolicy` resource does not restart the nodes.
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/images-configuration-registry-mirror.adoc:137]: Acked

File: modules/images-configuration-registry-mirror.adoc (hunk 1 above)
Comment (xenolinux) [reply modules/images-configuration-registry-mirror.adoc:132]: Acked

**Comment by xiuwang** [comment]
/lgtm
Thanks

**Comment by nalhadef** [comment]
Hi, Servesha. There's one small typo to fix. You can request a merge when that done. Pending the correction, ...

File: modules/images-configuration-registry-mirror.adoc [hunk 3]
```diff
@@ -77,17 +77,32 @@ spec:
...
+    source: registry.example.com <8>
 ----
 <1> Indicates the name of the image registry and repository.
-<2> Indicates the registry and repository containing the content that is mirrored.
-<3> You can configure a namespace inside a registry to use any image in that namespace. If you use a registry domain as a source, the `ImageContentSourcePolicy` resource is applied to all repositories from the registry.
+<2> Indicates multiple mirror repositories for each target repository. If one mirror is down, the target repository can use another mirror.
+<3> Indicates the registry and repository containing the content that is mirrored.
+<4> You can configure a namespace inside a registry to use any image in that namespace. If you use a registry domain as a source, the `ImageContentSourcePolicy` resource is applied to all repositories from the registry.
+<5> If you configure the registry name, the `ImageContentSourcePolicy` resource is applied to all repositories from a source registry to a mirror registry.
+<6> Pulls the image `mirror.example.net/image@sha256:...`.
```
Comment (mburke5678) [inline modules/images-configuration-registry-mirror.adoc:103]: For <6>, <7>,<8>, _image_ should come after the image name, I believe. For example:
_Pulls the `mirror.example.net/image@sha256:...`  image_

**Comment by mburke5678** [comment]
/enterprise-4.9

**Pull request merged**